tests/UI/campaigns/functional/FO/08_menuAndNavigation/03_navigationAndDisplay/02_pagination.ts

```diff
@@ -0,0 +1,264 @@
+// Import utils
+import helper from '@utils/helpers';
+import testContext from '@utils/testContext';
+
+// Import commonTests
+import loginCommon from '@commonTests/BO/loginBO';
+
+// Import pages
+import dashboardPage from '@pages/BO/dashboard';
+import homePage from '@pages/FO/home';
+import categoryPageFO from '@pages/FO/category';
+import productSettingsPage from '@pages/BO/shopParameters/productSettings';
+
+import {expect} from 'chai';
+import type {BrowserContext, Page} from 'playwright';
+
+const baseContext: string = 'functional_FO_menuAndNavigation_navigationAndDisplay_pagination';
+
+/*
+Scenario:
+- Go to FO>All products page
+- Check the pagination in the bottom of the page
+- Click on next then on previous
+- Edit products per page number to 6 in BO
+- Check the new pagination in FO
+- Edit products per page number to 20
+- Check the new pagination
+Post-condition:
+- Reset 'Number of products per page'
+ */
+describe('FO - Navigation and display : Pagination', async () => {
+  let browserContext: BrowserContext;
+  let page: Page;
+
+  // before and after functions
+  before(async function () {
+    browserContext = await helper.createBrowserContext(this.browser);
+    page = await helper.newTab(browserContext);
+  });
+
+  after(async () => {
+    await helper.closeBrowserContext(browserContext);
+  });
+
+  describe('FO - Pagination next and previous', async () => {
+    it('should open the shop page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'openShopPage', baseContext);
+
+      await homePage.goTo(page, global.FO.URL);
+
+      const result = await homePage.isHomePage(page);
+      await expect(result).to.be.true;
+    });
+
+    it('should go to all products page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToAllProducts', baseContext);
+
+      await homePage.changeLanguage(page, 'en');
+      await homePage.goToAllProductsPage(page);
+
+      const isCategoryPageVisible = await categoryPageFO.isCategoryPage(page);
+      await expect(isCategoryPageVisible, 'Home category page was not opened').to.be.true;
+    });
+
+    it('should check the number of products on the page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'numberOfProducts', baseContext);
+
+      const numberOfProducts = await categoryPageFO.getNumberOfProducts(page);
+      expect(numberOfProducts).to.eql(19);
+    });
+
+    it('should check the pagination in the bottom of the page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkPaginationLabel', baseContext);
+
+      const pagesList = await categoryPageFO.getPagesList(page);
+      await expect(pagesList).to.eq('1 2 Next');
```

Comment:
Normal that we have a square ?

Reply:
It's to verify the position of the next and previous icon 
![Capture d’écran de 2023-03-20 17-38-43](https://user-images.githubusercontent.com/24647495/226408377-86fb5455-f682-4ac9-a2ab-8cc01c94ccc9.png)

Reply:
Maybe not use square (it can change) and use `to.contain('1 2 Next')`

Reply:
Done :+1: